workflow/tests/hpxml_translator_test.rb

```diff
@@ -1194,7 +1197,9 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml)
 
     # Lighting
     ltg_energy = results.select { |k, v| k.include? 'End Use: Electricity: Lighting' }.map { |k, v| v }.sum(0.0)
-    assert_equal(hpxml.lighting_groups.size > 0, ltg_energy > 0)
+    if hpxml.header.schedules_path.nil?
+      assert_equal(hpxml.lighting_groups.size > 0, ltg_energy > 0)
+    end
```

Comment:
This change looks wrong; we should still check for lighting energy use in the simulations using schedule files too.